uiowa/uiowa
# [Facilities] Added coordinators view

Resolves https://github.com/uiowa/uiowa/issues/7361

# How to test

```
ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local fm-buildings && ddev drush @facilities.local uli node/1731/edit
```
- Look through buildings like https://facilities.uiowa.ddev.site/building/0001 and confirm primary and secondary coordinators are listed. 
  - Review other buildings with multiple secondary coordinators. 
- Empty view headers are still showing like on https://facilities.uiowa.ddev.site/building/0005
  - Review the no results settings and see if you can figure out why the view block is still rendering when empty?  https://github.com/uiowa/uiowa/issues/7368#issuecomment-1957044809
